tests/simulation_db_test.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+u"""test simulation_db operations
+
+:copyright: Copyright (c) 2020 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from __future__ import absolute_import, division, print_function
+import pytest
+
+def test_uid():
+    _do(
+        '/sim-db-file/user/xxx/elegant/RrCoL7rQ/flash_exe-SwBZWpYFR-PqFi81T6rQ8g',
+        'yyy',
+    )
+
+    _do(
+        '/sim-db-file/user/xxx/elegant/RrCoL7rQ/../../../foo',
+        'xxx',
+    )
+
+    _do(
+        '/sim-db-file/user/yyy/invalid/R/flash_exe-SwBZWpYFR-PqFi81T6rQ8g',
+        'yyy',
+    )
+
+    _do(
+        '/sim-db-file/user/yyy/invalid/RrCoL7rQ/flash_exe-SwBZWpYFR-PqFi81T6rQ8g',
+        'yyy',
+    )
+
+    _do(
+        '/sim-db-file/user/yyy/invalid/RrCoL7rQ/xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx',
```

Comment:
This is minor, but the length of the url might be better specified with a number so that we can see what limit is being tested. there are two illegal things going on "/yyy/invalid"  and the length. Probably best not to test them both in the same case.